### DISCUSSION TITLE
Uploading EVTX Files: URL dates swapped

### DISCUSSION THREAD

**Discussion by jwmasekre** [Security-Onion-Solutions/securityonion#11322] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

24

### Storage for /

65

### Storage for /nsm

126

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Discovered that when importing an EVTX to the manager via the grid page, the date range is swapped on the URL that's generated:
![image001 (5)](https://github.com/Security-Onion-Solutions/securityonion/assets/47164372/0170e033-96e2-4ba9-a6cd-94c12be09ae2)

Does this need to be submitted as an issue? it's minor but should probably be fixed for more junior users.

Tangentially related, I found that only my manager could ingest EVTX, which is contrary to the Grid documentation ("If your node is a network sensor or import node, then there will be an additional icon for uploading your own PCAP or EVTX file", which is technically true but it errors out of you try to upload an EVTX to a sensor).

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> Discovered that when importing an EVTX to the manager via the grid page, the date range is swapped on the URL that's generated:

I've confirmed a potential issue with certain EVTX files and created an issue:
https://github.com/Security-Onion-Solutions/securityonion/issues/11332

> Tangentially related, I found that only my manager could ingest EVTX, which is contrary to the Grid documentation ("If your node is a network sensor or import node, then there will be an additional icon for uploading your own PCAP or EVTX file", which is technically true but it errors out of you try to upload an EVTX to a sensor).

I've updated the documentation.
